spec/index.spec.mjs

```diff
@@ -7,7 +7,8 @@ import axiosRetry, {
   isIdempotentRequestError,
   exponentialDelay,
   isRetryableError
-} from '../es/index';
+  // eslint-disable-next-line import/extensions
+} from '../es/index.mjs';
```

Comment:
why we need this?

Reply:
It's a linter warning

Reply:
I don't see any linter issues that required a fix, neither we need to add the mjs extension here, given the project config